ui/src/components/SampleView/SampleControls.module.css

```diff
@@ -108,7 +108,6 @@
   margin-top: 0.5rem;
   padding: 0.625rem 0.75rem;
   background-color: rgba(0, 0, 0, 0.6);
-  color: rgb(255, 255, 255);
```

Comment:
Slightly unrelated: this fixes the text color of the input field of the "Focus" overlay on the sample view (which uses `MotorInput` via `OneAxisTranslationControl`).